File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p02755/p02755_num4_parsed.py
A, B = map(int, input().split())

for p in range(1, 101):
    tax8 = int(p * 0.08)
    tax10 = int(p * 0.1)
    if tax8 == A and tax10 == B:
        print(p)
        break
else:
    print(-1)

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p02755/p02755_num4_parsed.cpp
#include <iostream>
#include <cmath>

int main() {
    int A, B;
    std::cin >> A >> B;

    int price = -1;

    for (int i = 1; i <= 10000; i++) {
        int tax8 = std::floor(i * 0.08);
        int tax10 = std::floor(i * 0.1);

        if (tax8 == A && tax10 == B) {
            price = i;
            break;
        }
    }

    std::cout << price << std::endl;

    return 0;
}